packages/ocvImpact/R/utils_montagu.R

```diff
@@ -116,6 +116,13 @@ import_centralburden_template <- function(modelpath, country, cache, redownload
     rc <- readr::read_csv(paste0(modelpath, "/", fname)) %>%
       dplyr::filter(country == !!country) %>%
       dplyr::distinct(disease, country, country_name, year, age)
+    
+    ## if we are not using montagu coverage, only keep output years specified by the config -- this applies to the DRC Case study
```

Comment:
Add comment: When using montagu coverage, the output years are determined from the montagu coverage file